docs/lazy-loading.md

```diff
@@ -0,0 +1,45 @@
+# Lazy Loading
+
+The Media module supports Lazy Loading of images using low resolution image placeholders with a blur effect, generating accurate preview images using an amazingly small amount of data.
+
+This is implemented using the [Gaussholder](https://github.com/humanmade/gaussholder) project.
+
+Gaussholder takes a Gaussian blur and applies it to an image to generate a preview image. Gaussian blurs work as a low-pass filter, allowing us to throw away a lot of the data. We then further reduce the amount of data per image by removing the JPEG header and rebuilding it on the client side (this eliminates ~800 bytes from each image).
+
+## Configuration
+
+Lazy loading of images in post content is enabled by default. This behavior can be disabled by setting the `modules.media.gaussholder` property to `false` in your project's `composer.json`.
+
+Lazy loading is enabled on a per image size basis, so you must configure the specific image sizes. This is done via the `image-sizes` key in the configuration:
+
+```json
+"extra": {
+	"platform": {
```

Comment:
Indentation in this block is off.

Reply:
Hmm where?

Reply:
Aha, GitHub rendering bug with tabs. Disregard!